x-pack/plugins/graph/public/app.js

```diff
@@ -730,30 +759,46 @@ app.controller('graphuiPlugin', function ($scope, $route, $interval, $http, kbnU
   $scope.topNavMenu = [];
   $scope.topNavMenu.push({
     key: 'new',
-    description: 'New Workspace',
-    tooltip: 'Create a new workspace',
+    description: i18n('xpack.graph.topNavMenu.newWorkspaceAriaLabel', {
+      defaultMessage: 'New Workspace',
+    }),
+    tooltip: i18n('xpack.graph.topNavMenu.newWorkspaceTooltip', {
+      defaultMessage: 'Create a new workspace',
+    }),
     run: function () {canWipeWorkspace(function () {kbnUrl.change('/home', {}); });  },
   });
   if (!$scope.allSavingDisabled) {
     $scope.topNavMenu.push({
       key: 'save',
-      description: 'Save Workspace',
-      tooltip: 'Save this workspace',
+      description: i18n('xpack.graph.topNavMenu.saveWorkspace.enabledAriaLabel', {
+        defaultMessage: 'Save Workspace',
+      }),
+      tooltip: i18n('xpack.graph.topNavMenu.saveWorkspace.enabledTooltip', {
+        defaultMessage: 'Save this workspace',
+      }),
       disableButton: function () {return $scope.selectedFields.length === 0;},
       template: require('./templates/save_workspace.html')
     });
   }else {
     $scope.topNavMenu.push({
       key: 'save',
-      description: 'Save Workspace',
-      tooltip: 'No changes to saved workspaces are permitted by the current save policy',
+      description: i18n('xpack.graph.topNavMenu.saveWorkspace.disabledAriaLabel', {
+        defaultMessage: 'Save Workspace',
+      }),
+      tooltip: i18n('xpack.graph.topNavMenu.saveWorkspace.disabledTooltip', {
+        defaultMessage: 'No changes to saved workspaces are permitted by the current save policy',
+      }),
       disableButton: true
     });
   }
   $scope.topNavMenu.push({
     key: 'open',
-    description: 'Load Saved Workspace',
-    tooltip: 'Load a saved workspace',
+    description: i18n('xpack.graph.topNavMenu.loadWorkspaceAriaLabel', {
```

Comment:
AriaLabel => Description. But if it is used for aria-label attribute, then it is okay, it is AriaLabel